src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let result = 0;
+
+  function adder(num) {
+    if (num !== undefined) {
+      result += num;
+
+      return adder;
+    } else {
+      const finalResult = result;
```

Comment:
Not critical in this task, but it would be great to use more meaningful names than just `result` and `finalResult`